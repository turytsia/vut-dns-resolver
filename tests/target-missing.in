-r -t -s kazi.fit.vutbr.cz