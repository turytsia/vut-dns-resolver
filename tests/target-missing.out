Error: Target address is not specified
